### ISSUE TITLE
XSeg training stops after loading all samples (no errors)

### ISSUE THREAD

**Issue by archievawser** [iperov/DeepFaceLab#5736]
## Expected behavior

Attempting to train XSeg by running `5.XSeg) train.bat` after generating masks using the default generic XSeg model.

After the XSeg trainer has loaded samples, it should continue on to the filtering stage and then begin training.

## Actual behavior

Instead of the trainer continuing after loading samples, it sits idle doing nothing infinitely like this:

![image](https://github.com/iperov/DeepFaceLab/assets/94964623/6eecad63-1f62-42b3-b761-bf414a5eaaaa)

There are no errors visibly being logged.

## Steps to reproduce

1. Run `5.XSeg) train.bat`
2. Select NVIDIA GeForce RTX 2080 SUPER GPU
3. Use whole face type
4. Use a batch size of 8
5. Do not enable pretraining mode

## Other relevant information
**OS:** Windows 10
**GPU:** NVIDIA GeForce RTX 2080 SUPER
**CPU:** AMD Ryzen 9 3900X
**RAM:** 16GB

I've tried modifying virtual ram and changing drivers.
